core/controllers/profile.py

```diff
@@ -108,6 +117,8 @@ def get(self, username):
                 user_settings.user_id),
             'created_exp_summary_dicts': created_exp_summary_dicts,
             'edited_exp_summary_dicts': edited_exp_summary_dicts,
+            'created_collection_summary_dicts': created_collection_summary_dicts,    #pylint: disable=line-too-long
```

Comment:
We want to reduce the number of lines that disable pylint checks, so it would be better if you renamed the variable so that it is shorter.